lib/venue.rb

```diff
@@ -1,2 +1,47 @@
 class Venue
+  attr_accessor :bird_type, :capacity, :capacity
+
+  def initialize(bird_type, capacity)
+    @bird = bird_type
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def name
+    @bird
+  end
+
+  def capacity
```

Comment:
You have `capacity` in your `attr_accessor` so you would not need to create a method here. That is what is happening under the hood in the `attr_accessor`